File: bubblesort.cpp
#include<iostream>
#include<bits/stdc++.h>
#include<chrono>
using namespace std;
using namespace std::chrono;
int main() {
    auto start = high_resolution_clock::now();
    int arr[6] = {12,45,23,51,19,8};
    for (int i =1;i<6;i++)
    {
        for(int j = 0;j<6-i;j++)
        {
            if(arr[j+1]<arr[j]) swap(arr[j+1],arr[j]);
        }
    }
//    for (auto elem:arr)
//     {
//         cout<<elem<<" ";
//     }
//     cout<<"\n";









    auto stop = high_resolution_clock::now();
    auto duration = duration_cast<microseconds>(stop - start);
 
    cout << "Time taken by bubble sort: "
         << duration.count() << " microseconds" << endl;
}

File: bubblesort.py
import time
import time
lis = [12,45,23,51,19,8]
then = time.time()
for i in range(1,6):
    for j in range(0,6-i):
        if (lis[j+1]<lis[j]):
            temp = lis[i]
            lis[i] = lis[j]
            lis[j] = temp
print(lis)
now = time.time()
print("Time taken by bubble sort:",(now - then)*(10**6)," microseconds" )